## Tool 1: Pre-Training (Crosstab Tool)

We begin by checking for proxy bias between two columns `location_type` and `gender`.
Using my Crosstab Tool, If either male or female respondants in rural or urban locations
are above my 55% threshold, this confirms `location_type` 
is a proxy/subsitute for `gender`. In other words, `gender` is baked/hidden/camoflauged inside of `location_type`.
This is called a hidden discrimination risk.
Note that the 55% threshold mark I set is not a universal standard. 
I set it as a threshold just for this specific audit.

In [16]:
# Create the distribution
proxy_report = pd.crosstab(df['location_type'], df['gender_of_respondent'], normalize='index')

# Add the 'Total' column for clarity
proxy_report['Total'] = 1.0 

# Format as a professional audit table with a warning color for anything over 55%
proxy_report.style.format("{:.2%}")\
    .background_gradient(cmap='Reds', subset=['Female','Male'], low=0.4, high=0.6)\
    .set_caption("Table 1.1: Location x Gender Proxy Risk Assessment")

gender_of_respondent,Female,Male,Total
location_type,,,
Rural,59.50%,40.50%,100.00%
Urban,58.20%,41.80%,100.00%


## Finding 1: Proxy Bias Confirmed

**Rural respondents are 59.50% female vs 40.50% male.**
**Urban respondents are 58.20% female vs 41.80% male.**

Female respondents at both Rural and Urban locations exceed my 55% threshold. This confirms that `location_type` is a proxy variable for `gender_of_respondent`. If the model is heavily weighted on `location_type`, it will indirectly discriminate by gender, without ever explicitly using gender as an input. This is called **hidden discrimination risk** or **proxy bias**.

**Note:** The 55% threshold is a ClearBoxAI internal standard set specifically for this audit. It is not a universal regulatory requirement. No precise numerical threshold for proxy bias detection is specified in BoG CISD 2026, the EU AI Act, or the NIST AI RMF.

---

**Risk Level: HIGH**

| Regulatory Framework | Provision | Relevance |
|---|---|---|
| BoG CISD 2026 | Section 100(2)(b) | AI/ML Oversight Committee must conduct ethical and fairness assessments |
| BoG CISD 2026 | Annexure E | Names "Bias and Discrimination: Risks of unfair outcomes due to biased training data or model" explicitly |
| EU AI Act | Article 10(2)(f) | Training data must be examined for "possible biases likely to lead to discrimination" |
| NIST AI RMF | MEASURE 2.11 | "Fairness and bias are evaluated and results are documented" |

---

**Auditor:** Kwadwo Amponsah | ClearBoxAI
**Date:** April 2026